doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+### Use case: Rename tag of person
+
+**MSS**
+
+1. User requests to rename tag of a specific person
+2. AddressBook shows confirmation prompt
+3. User confirms request to rename tag
+4. AddressBook renames the existing tag with a new tag
+Use case ends.
+
+**Extensions**
+
+2a. No person with the tag
```

Comment:
`1. User requests to rename tag of a specific person` & `No person with the tag` -> The description doesn't seems to match each other.